SwiftLeeds/Views/Common/CommonCompo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CommonCompo.swift
+//  SwiftLeeds
+//
+//  Created by LUCKY AGARWAL on 23/07/22.
+//
+
+import Foundation
+import SwiftUI
+
+class CommonCompo {
+    
+    static func sectionHeader(text: String) -> some View {
+        Text(text)
+            .font(.callout.weight(.semibold))
+            .foregroundColor(.secondary)
+            .frame(maxWidth:.infinity, alignment: .leading)
```

Comment:
Can we throw a space between maxWidth: and .infinity to match the rest of the line?

Reply:
Got it